src/app/components/App.tsx

```diff
@@ -284,16 +285,31 @@ const BackToTheTop = lazy(() =>
 );
 
 // Helper component to render a landing page or the results page depending on
-// the presence of absence of a querystring
+// the presence or absence of a query string (any URL parameters) eg presence
+// of any of: facets, query, sort will be routed to a results page. In the case
+// of query string is present but no "query=" or this is empty it will be
+// provided as "query=*"
 const ResultsOrLanding =
   (ResultsPage: FC<RouteChildrenProps>, LandingPage: FC<RouteChildrenProps>) =>
-  (props: RouteChildrenProps) =>
-    props.location.search ? (
-      <ResultsPage {...props} />
-    ) : (
-      <LandingPage {...props} />
-    );
-
+  (props: RouteChildrenProps) => {
+    if (props.location.search) {
+      const params = Object.fromEntries(
+        new URLSearchParams(props.location.search)
+      );
+      if (!params.query) {
+        return (
+          <Redirect
+            to={queryString.stringifyUrl({
+              url: props.location.pathname,
+              query: { ...params, query: '*' },
```

Comment:
I didn't know this is coming. I added a specific query=* targeting one thing here - https://github.com/ebi-uniprot/uniprot-website/commit/b9ab0e073118e8bc9b5b84508fce31c97071cc90

I tested by reverting it and the results page load with query=*. I will commit (undoing my previous fix) in this PR if you don't mind